packages/flutter_tools/test/general.shard/android/gradle_errors_test.dart

```diff
@@ -790,6 +790,34 @@ Execution failed for task ':app:generateDebugFeatureTransitiveDeps'.
       FileSystem: () => fileSystem,
       ProcessManager: () => processManager,
     });
+
+    testUsingContext('generates correct gradle command for Unix-like environment', () async {
+      await lockFileDepMissingHandler.handler(
+        project: FlutterProject.fromDirectoryTest(fileSystem.currentDirectory),
+        usesAndroidX: true,
+        line: '',
+      );
+      expect(testLogger.statusText, contains('./gradlew'));
```

Comment:
Can we match on more than just the name of the file, maybe include the text before

`To regenerate the lockfiles run:...`